pkg/apis/apiserver/organization_REST.go

```diff
@@ -19,25 +19,50 @@ package apiserver
 import (
 	"context"
 	"fmt"
+	"net/http"
 
-	v1 "k8s.io/api/core/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/api/meta"
 	"k8s.io/apimachinery/pkg/apis/meta/internalversion"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
 	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/watch"
+	"k8s.io/apiserver/pkg/endpoints/filters"
 	"k8s.io/apiserver/pkg/registry/generic"
 	"k8s.io/apiserver/pkg/registry/rest"
 	"k8s.io/client-go/dynamic"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 	"sigs.k8s.io/controller-runtime/pkg/runtime/inject"
+
+	corev1alpha1 "github.com/kubermatic/bulward/pkg/apis/core/v1alpha1"
+)
+
+const (
+	internalOrganizationResource = "organizations"
+	externalOrganizationResource = "organizations"
+)
+
+var (
+	qualifiedResource = schema.GroupResource{
+		Group:    SchemeGroupVersion.Group,
+		Resource: externalOrganizationResource,
+	}
 )
 
 // +k8s:deepcopy-gen=false
 type OrganizationREST struct {
-	client  client.Client
-	dynamic dynamic.Interface
-	mapper  meta.RESTMapper
-	scheme  *runtime.Scheme
+	client    client.Client
```

Comment:
This client is not used anywhere I think?

Reply:
it will be...and it doesn't hurt that much, does it?